### PR TITLE
prepare 2.0.0 release

### PR DESCRIPTION
**Pull request by owaddell-beckman**
Still a fair bit to do here:
- [X] sketch GitHub issue template (have folks include output of `swish-build --info`)
- [X] sketch GitHub pull-request template
- [X] create dummy private repo to test drive issue / pull-request templates
- [X] get Bob's .astylerc update
- [X] remember to snag issue / pull-request template changes from dummy repo
- [X] draft some Contributing guidelines
- [X] add blurb about semantic versioning in ReadMe
- [X] mark ready for review, without rebasing
- [x] try running src/run-manual-test on Windows
- [ ] rebase for clarity
- [ ] ?

### CODE REVIEW

**Comment by burgerrg** [comment]
src/run-manual-test does this if all I've done is `./configure` and `make -C src/swish mat-prereq`:

```
DESKTOP-8N36EVU:~/swish$ src/run-manual-test 1
== Case 1
src/run-manual-test: line 25: data/tmp/manual-test-case: No such file or directory
manual-test-case: I/O error -4058 from uv_fs_realpath on data/tmp/manual-test-case: no such file or directory.
```

The script assumes the data/tmp directory has already been created.

**Comment by owaddell-beckman** [comment]
Good catch. Pushed an attempted fix.

File: doc/swish/app.tex [hunk 1]
```diff
@@ -82,21 +82,24 @@
 To provide arguments to the Swish executable before the script filename, add
 the \sopt{S} option to \code{env} and add the desired arguments after
 \code{swish}.
-Here the \lopt{} option tells Swish to start a REPL after loading the script.
-\codebegin
+Here the \code{-q} option suppresses the startup message and sets the prompt to
+the empty string and the \lopt{} option tells Swish to start a REPL after
```
Comment (burgerrg) [inline doc/swish/app.tex]: suggestion: add a comma between `string` and `and`.

**Review by burgerrg — COMMENTED** [review]
That fixed src/run-manual-test, thanks!

**Comment by owaddell-beckman** [comment]
Sorry, I flipped it from a draft pull request prematurely. I think the changes I meant to include are now in place.

**Review by burgerrg — APPROVED** [review]
Great work, Oscar, thanks!

**Comment by owaddell-beckman** [comment]
Pushed an attempt at rebase cleanup. I wouldn't mind squashing in the .gitignore fix to make it tidier when we tag the release.

**Comment by owaddell-beckman** [comment]
Pushed some minor documentation updates to include in 2.0.0. I plan to rebase these ahead of the "releae: swish 2.0.0" commit before we push the big red button.

**Comment by owaddell-beckman** [comment]
Went ahead and squashed the .gitignore fix and reordered a few commits since I wanted to update some of the log messages.

**Comment by owaddell-beckman** [comment]
Thanks, Bob and Chris, for your help in getting this release out the door.

**Pull request merged**